### DISCUSSION TITLE
暗水印那几个配置项就只有一种配置吗？

### DISCUSSION THREAD

**Discussion by 6ddds1** [zhensherlock/watermark-js-plus#86] [Q&A] [answered]
暗水印那几个配置项就只有一种配置吗？

**Comment by zhensherlock** [comment]
没有理解，配置项是什么？

**Reply by 6ddds1** [reply]
比如说mode  就只有canvas吗？我这边透明度设置50%以上后，暗水印好像没生效，解析不出来。

**Reply by zhensherlock** [reply, marked as answer]
mode可能得值是'default' | 'blind'，并没有‘canvas’；修改globalAlpha参数并不会生效，实例化时会自动改成0.005；解析的时候，需要注意区分主题颜色，黑色主题的图片需要使用Dark Background解析图片

**Reply by zhensherlock** [reply]
哦，你说的mode是Blind Watermark Decode的参数是吗？目前解析只实现了canvas的方式。
